src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt

```diff
@@ -22,6 +22,9 @@ data class DataSources(
      *  If index is pre-existing, mapping is updated. */
     val alertsIndex: String = ".opendistro-alerting-alerts", // AlertIndices.ALERT_INDEX
 
+    /** Configures a custom index to store historic alerts for a monitor.*/
+    val alertsHistoryIndex: String,
```

Comment:
let's add the default value used in alerting plugin

Reply:
done